tests/test_standalone_stac_item.py

```diff
@@ -36,11 +36,11 @@ def test_standalone_stac_item_thredds_ncml():
     # FIXME: avoid hackish workarounds
     data = requests.get(thredds_ncml_url).text
     attrs = xncml.Dataset.from_text(data).to_cf_dict()
-    attrs["access_urls"] = {  # FIXME: all following should be automatically added, but they are not!
```

Comment:
Comment removed because they aren't automatically added and they aren't expected to be. See the `extract_metadata` function.

Reply:
The fix needs to be done in siphon then, not here. A `# FIXME` comment implies that this needs to be fixed in this code-base.